### ISSUE TITLE
Session Keys: UX Update to Disable Actions

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#4317]
## User Story
- As a user, I need to know the actions I can and cannot take when I haven’t joined a community with my current “connected” (primary) address, so I can decide whether I want to switch to an address I already have linked to the community or to join the community.

**Solution to Implement:**
- Disable actions for users who’s address haven’t “joined” a community
  - Disable the upvote functionality
  - Disable the comment functionality (should be already done).

This ticket is closely related to #4378.

**Comment by CowMuon** [comment]
Some questions here around relations between a linked address and being joined to a community. Just discussed with Jake, sounds like we will need Zak's input to clarify.

**Comment by masvelio** [comment]
From my investigation it looks that we only need to disable big upvote button 

![Image](https://github.com/hicommonwealth/commonwealth/assets/14819225/25ffb7c6-4b0a-4dc0-92cf-267dcb9aff7a)

Blocking other actions has been already done in https://github.com/hicommonwealth/commonwealth/pull/4521

**Comment by masvelio** [comment]
In this ticket I am gonna also take care about displaying revalidation-session-modal properly so it would be the last step before we can move session-keys to the QA (at least from my perspective)

**Comment by masvelio** [comment]
Work has been pushed to `raymond.react-session-keys-on`: 
- disabling button https://github.com/hicommonwealth/commonwealth/pull/3426/commits/45c0c02413df3bd74ad05658ff1f80bfc409de56
- moving RevalidationSessionModal to reusable hook and use that hook in multiple places (there are more commits but this is most important) https://github.com/hicommonwealth/commonwealth/pull/3426/commits/dd8622d389206b1f5b0ebeeac05910ab3abd4f3a

Because we are working on feature-branch, I am closing this ticket
